apps/nextjs/tests-e2e/tests/demo-accounts.test.ts

```diff
@@ -0,0 +1,22 @@
+import { clerkSetup, setupClerkTestingToken } from "@clerk/testing/playwright";
+import { test, expect } from "@playwright/test";
+
+import { TEST_BASE_URL } from "../config/config";
+import { prepareUser } from "../helpers/auth";
+import { bypassVercelProtection } from "../helpers/vercel";
+
+test.describe("Demo accounts", () => {
+  test("should see the demo banner", async ({ page }) => {
+    await bypassVercelProtection(page);
+    await clerkSetup();
+    await setupClerkTestingToken({ page });
+
+    await prepareUser(page, "demo");
+
+    await page.goto(`${TEST_BASE_URL}/aila`);
+    await expect(page.getByTestId("demo-banner")).toBeVisible();
+    await expect(page.getByTestId("demo-banner")).toContainText(
+      "Create 3 lessons per month",
```

Comment:
Maybe just "lessons per month" if that is going to change?

Reply:
That's an interesting point. Ideally it should be reset when the user is prepared so this is actually quite a useful test!